README.md

```diff
@@ -130,7 +130,7 @@ with sync_playwright() as playwright, playwright.chromium.launch(headless=False)
 - [Save and reuse logged in state](https://github.com/tinyfish-io/agentql/tree/main/examples/save_and_load_authenticated_session)
 - [Run AgentQL in Stealth Mode](https://github.com/tinyfish-io/fish-tank/tree/main/examples/stealth_mode)
 - [Wait for page to load](https://github.com/tinyfish-io/agentql/tree/main/examples/wait_for_entire_page_load)
-- [Collect pricing data from an e-commerce website using AgentQL](https://github.com/tinyfish-io/fish-tank/tree/main/examples/close_cookie_dialog)
+- [Collect pricing data from an e-commerce website using AgentQL](https://github.com/tinyfish-io/agentql/tree/main/application_examples/collect_ecommerce_pricing_data)
 - [Perform sentiment analysis](https://github.com/tinyfish-io/fish-tank/tree/main/application_examples/perform_sentiment_analysis)
 - [Get an element's XPath](https://github.com/tinyfish-io/fish-tank/tree/main/application_examples/xpath)
 - [Run script online in Google Colaboratory](./examples/run_script_online_in_google_colab)
```

Comment:
I think this (using relative links) might actually be a better idea in terms of being prone to renames, etc.

Reply:
Thanks for catching these! You make a good case. I endorse it for a future update